af/fastpath/fastpath/db.py

```diff
@@ -182,24 +184,25 @@ def tf(v: bool) -> str:
         msm["measurement_start_time"], "%Y-%m-%d %H:%M:%S"
     )
     test_start_time = datetime.strptime(msm["test_start_time"], "%Y-%m-%d %H:%M:%S")
-    row = [
-        measurement_uid,
-        nn(msm, "report_id"),
-        input_,
-        nn(msm, "probe_cc"),
-        asn,
-        test_name,
-        test_start_time,
-        measurement_start_time,
-        ujson.dumps(scores),
-        nn(msm, "platform"),
-        tf(anomaly),
-        tf(confirmed),
-        tf(msm_failure),
-        domain,
-        nn(msm, "software_name"),
-        nn(msm, "software_version"),
-    ]
+    row = dict(
+        measurement_uid=measurement_uid,
+        report_id=nn(msm, "report_id"),
+        input=input_,
+        probe_cc=nn(msm, "probe_cc"),
+        probe_asn=asn,
+        test_name=test_name,
+        test_start_time=test_start_time,
+        measurement_start_time=measurement_start_time,
+        scores=ujson.dumps(scores),
+        platform=nn(msm, "platform"),
```

Comment:
This should be reading the value of the keyword argument `platform` being passed to the function instead of the key in the measurement.

Reply:
This looks like a regression bug that was introduced during the migration to clickhouse, so it's not something new. In postgres we were extracting it properly: https://github.com/ooni/pipeline/blob/master/af/fastpath/fastpath/db.py#L131 and I confirmed we have this always set to the empty string in the DB.

Reply:
This is unrelated to the current PR. Anyhow we are currently using "" in the database for unset platform. I don't see much point in switching to "unset" as a string at this point. I can remove the logic in core.py